clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -43,7 +43,7 @@
     private Cluster cluster;
     private boolean needUpdate;
     private final Set<String> topics;
-    private final List<Listener> listeners;
+    private final Set<Listener> listeners;
```

Comment:
Yeah, my fault. Seemed logical to be a ``Set`` instead of a ``List`` so that we don't have the same listener duplicated. But I need to stop doing this kind of unrelated change. Excuse me. :) Gonna revert now.